src/app/EventLoggingTypes.h

```diff
@@ -128,14 +128,20 @@ class EventOptions
 {
 public:
     EventOptions() : mPriority(PriorityLevel::Invalid) {}
-    EventOptions(Timestamp aTimestamp) : mTimestamp(aTimestamp), mPriority(PriorityLevel::Invalid) {}
     ConcreteEventPath mPath;
-    Timestamp mTimestamp;
     PriorityLevel mPriority = PriorityLevel::Invalid;
     // kUndefinedFabricIndex 0 means not fabric associated at all
     FabricIndex mFabricIndex = kUndefinedFabricIndex;
 };
 
+class InternalEventOptions : public EventOptions
```

Comment:
So is this public API?  If not, why is it sitting next to public API bits?  We need to be much better about which things are public and which are not.

Reply:
https://github.com/project-chip/connectedhomeip/pull/38290